src/Sulu/Component/Content/Tests/Functional/Repository/ContentRepositoryTest.php

```diff
@@ -637,11 +637,32 @@ public function testFindWithNonFallbackProperties()
             )->getMapping()
         );
 
-        $this->assertGreaterThan($link->getCreated(), $result['created']);
-        $this->assertGreaterThan($link->getChanged(), $result['changed']);
+        $created = $result['created'];
+        $changed = $result['changed'];
 
-        $this->assertEquals($page->getChanged(), $result['changed']);
-        $this->assertEquals($page->getCreated(), $result['created']);
+        // Jackalope Jackrabbit will return a \DateTime and DBAL will return a
+        // string. See: https://github.com/jackalope/jackalope-doctrine-dbal/issues/325
+        if (is_string($created)) {
+            $created = new \DateTime($result['created']);
+        }
+
+        if (is_string($changed)) {
+            $changed = new \DateTime($result['changed']);
+        }
+
+        $this->assertGreaterThan($link->getCreated(), $created);
+        $this->assertGreaterThan($link->getChanged(), $changed);
+
+        // Jackalope Doctrine DBAL will currently "normalize" to UTC when
+        // persisting, but load in the default TZ on loading - normalizing to
+        // UTC means the dates are equivilent but not the same for the purposes
+        // of comparison.
+        //
+        // Pending PR: https://github.com/jackalope/jackalope-doctrine-dbal/pull/326
+        $page = $this->documentManager->find($page->getUuid(), 'de');
```

Comment:
Also, explain why we reload (because reloading will cause the date to be updated to the correct value).